Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocColumnRendererBase.cs

```diff
@@ -305,20 +322,31 @@ void IBocColumnRenderer.RenderDataCell (BocColumnRenderingContext renderingConte
     /// This is a template method. Deriving classes must implement <see cref="RenderCellContents"/> to provide the contents of
     /// the table cell (&lt;td&gt;) element.
     /// </remarks>
-    protected virtual void RenderDataCell (
+    private void RenderDataCell (
         BocColumnRenderingContext<TBocColumnDefinition> renderingContext,
         in BocDataCellRenderArguments arguments)
+    {
+      var bocCellAttributeRenderingContext = new BocCellAttributeRenderingContext<TBocColumnDefinition>(renderingContext);
+      AddAttributesToRenderForDataCell(bocCellAttributeRenderingContext, in arguments);
+      renderingContext.Writer.RenderBeginTag(HtmlTextWriterTag.Td);
```

Comment:
strategic blank line
```suggestion
      var bocCellAttributeRenderingContext = new BocCellAttributeRenderingContext<TBocColumnDefinition>(renderingContext);
      AddAttributesToRenderForDataCell(bocCellAttributeRenderingContext, in arguments);

      renderingContext.Writer.RenderBeginTag(HtmlTextWriterTag.Td);
```